src/Donations/PaymentMethods/NewPaypal.tsx

```diff
@@ -67,11 +67,28 @@ function NewPaypal({
   }
 
   const onError = (data) => {
-    setPaymentError(`Your order ${data.orderID} failed due to some error.`);
+    setPaymentError(`Your order failed due to some error.`);
+
+    // This function shows a transaction success message to your buyer.
```

Comment:
Why does the comment is talking about a `transaction success message` if something failed?